Packs/CortexXpanse/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Cortex Xpanse",
     "description": "Content for working with Attack Surface Management (ASM).",
     "support": "xsoar",
-    "currentVersion": "1.0.19",
+    "currentVersion": "1.1.0",
```

Comment:
it's minor change, it should be 1.0.20, and please change the rn file name as well.

Reply:
I don't believe I'm the one who changed that file, I think it's auto-updated by GitHub or the contribution wizard. If there's something I can do on my end to change the version number, let me know.

Reply:
its okay, it just should be 1.0.20 instead and the name of the release notes files should be 1_0_20.md